src/main/java/auth/controller/AppController.java

```diff
@@ -0,0 +1,62 @@
+package auth.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import auth.model.User;
+import auth.service.UserService;
+import auth.validator.UserValidator;
+
+@Controller
+@RequestMapping("/")
+public class AppController {
+
+	@Autowired
+	private UserService userService;
+
+	@Autowired
+	private UserValidator userValidator;
+
+	@RequestMapping("/login")
+	public String loginPage(Model model) {
+		return "login";
+	}
+
+	@RequestMapping("/")
+	public String homePage(Model model) {
+		return "home";
+	}
+
+	@RequestMapping(value = "/registration", method = RequestMethod.GET)
+	public String createAccount(Model model) {
+		model.addAttribute("user", new User());
+		return "registration";
+	}
+
+	@RequestMapping(value = "/registration", method = RequestMethod.POST)
+	public String doCreateAccount(@ModelAttribute("user") User user, BindingResult bindingResult, Model model) {
+		userValidator.validate(user, bindingResult);
```

Comment:
The UserValidator can be registered on dataBinder (http://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/web/bind/annotation/InitBinder.html) so you will not be needed to call the validate method.